tests/queries/0_stateless/00417_kill_query.sh

```diff
@@ -7,19 +7,26 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
-QUERY_FIELND_NUM=4
+QUERY_FIELD_NUM=4
 
-$CLICKHOUSE_CLIENT --max_block_size=1 -q "SELECT sleep(1) FROM system.numbers LIMIT 30" &>/dev/null &
-sleep 1
```

Comment:
One second can be not enough for a query to start (and sleep is not a proper way of synchronization).